lib/perl/Genome/Model/ReferenceAlignment/Command/GenotypeMicroarrayConcordance.pm

```diff
@@ -19,10 +19,16 @@ class Genome::Model::ReferenceAlignment::Command::GenotypeMicroarrayConcordance
             doc => 'The minimum depth required to consider concordant.',
             example_values => ['4'],
         },
+        bedtools_version => {
+            is => 'Text',
+            doc => 'Version of bed tools intersect to use.',
+            default_value => Genome::Model::Tools::BedTools->latest_bedtools_version,
+            valid_values => [ Genome::Model::Tools::BedTools->available_bedtools_versions ],
+        },
         picard_version => {
             is => 'Text',
             doc => 'The version of picard to use.',
-            example_values => ['1.123'],
+            valid_values => [ Genome::Model::Tools::Picard->available_picard_versions ],
```

Comment:
I believe this tool was added to Picard in `1.122`, so many of these won't be valid versions for this tool.